src/Engine/Spells/CastSpellInfo.cpp

```diff
@@ -3360,28 +3360,28 @@ void _42777D_CastSpell_UseWand_ShootArrow(SPELL_TYPE spell,
             case SPELL_SPIRIT_BLESS:
                 if (!skill_value)
                     skill_value = player->pActiveSkills[PLAYER_SKILL_SPIRIT];
-                if (GetSkillMastery(skill_value) >= PLAYER_SKILL_MASTERY_EXPERT)
+                if (GetSkillMastery(skill_value) < PLAYER_SKILL_MASTERY_EXPERT)
                     flags |= ON_CAST_WholeParty_BigImprovementAnim;
                 break;
```

Comment:
Does this flag need renaming then? Has its use been mistakenly inverted elsewhere?

Reply:
In actual spell casting function mastery is used correctly.
As for renaming I think all of these ON_CAST_* flags need renaming but for now I have no idea how to call them correctly.

Reply:
Probably worth changing to something like "ON_CAST_SinglePlayer_BigImprovementAnim" for now and then leaving a TODO for renaming ON_CAST_ flags later.